python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -706,7 +720,7 @@ def _generate_images(
                     f"Writing Imsim Configuration file to {imsim_config_path}"
                 )
                 self.imsim_cmpt.write_yaml_and_run_imsim(
-                    imsim_config_path, imsim_config_yaml
+                    imsim_config_path, imsim_config_yaml, imsim_log_file=imsim_log_file
```

Comment:
Maybe it would be nice to be consistent and write this line in three lines? The same for line 682 in this file

Reply:
This formatting is set by `black` which all T&S code uses.